paasta_tools/contrib/paasta_update_soa_memcpu.py

```diff
@@ -214,11 +223,14 @@ def main(argv=None):
 
         serv['state'] = provisioned_state
         ticket_desc = (
-            "We suspect that {s}.{i} in {c} may be {o}-provisioned"
-            " as of {d}. It initially had {x} cpus, but we think"
+            "This ticket and CR have been auto-generated, and only requires action from your team to ship."
```

Comment:
What action? Just say "All you need to do is give a shipit if you agree, then X"
Where X is... what actually happens. Will it get merged immediately ? Does a perf team member merge it in in a week? Just say what happens to give developers more faith in what they are shipping.

Reply:
I have been accused of talking diagonally before... thanks for the second round of feedback.

I'm hoping that everyone will merge this themselves after our and their review. I don't want to make anyone feel like we're executive decisioning their service configuration.

New diff is up with another version of this ticket description.